src/sdk/sql_cluster_router.cc

```diff
@@ -1475,5 +1475,21 @@ ::openmldb::nameserver::TableInfo SQLClusterRouter::GetTableInfo(const std::stri
     return *table_infos;
 }
 
+::openmldb::base::Status SQLClusterRouter::ShowJobs(const bool only_unfinished,
+                                                    std::vector<::openmldb::taskmanager::JobInfo>& job_infos) {
+    auto taskmanager_client_ptr = cluster_sdk_->GetTaskManagerClient();
+    return taskmanager_client_ptr->ShowJobs(only_unfinished, job_infos);
```

Comment:
what if shared_ptr ref nullptr?

Reply:
We will add checking for pointer before using.